analyses/bsve/search/minerva.json

```diff
@@ -0,0 +1 @@
+{ "minerva": { "analysis_type": "bsve_search" } }
```

Comment:
Rather than having a separate file for the analysis type, why don't we just use the analysis name which is already contained in the analysis JSON?